projects/admin/src/app/migration/deduplication/record/brief-view/migration-data-deduplication/migration-data-deduplication.component.html

```diff
@@ -18,64 +18,91 @@
 @if (record()) {
 <div class="flex w-100 justify-content-between mb-2">
   <div class="flex align-items-center justify-content-center">
-    <p-inplace #inplace [active]="isInplaceActive">
+    <p-inplace #inplace>
       <ng-template pTemplate="display">
-        <strong>ILS pid:</strong> {{ ilsPid }}&nbsp;<i
-          class="pi pi-pencil"
-        ></i>
+        <strong>ILS pid:</strong> {{ ilsPid }}&nbsp;<i class="pi pi-pencil"></i>
```

Comment:
`{{ ilsPid }} <i class="pi pi-pencil ml-1"></i>` ?
One `&nbsp;` equal to one space